pep-0572.rst

```diff
@@ -630,8 +654,10 @@ Broadly the same semantics as the current proposal, but spelled differently.
    programmable calculators. (Note that a left-facing arrow ``y <- f(x)`` is
    not possible in Python, as it would be interpreted as less-than and unary
    minus.) This syntax has a slight advantage over 'as' in that it does not
-   conflict with ``with`` and ``except`` statements, but otherwise is
-   equivalent.
+   conflict with ``with``, ``except`` and ``import``, but otherwise is
+   equivalent.  But it is entirely unrelated to Python's other use of
+   ``->`` (function return type annotations), and compared to ``:=``
+   (which dates back to Algol-58) it has a much weaker tradition.
```

Comment:
I'm not sure `->` had enough advocates to be worth the bother of "refuting" - but since you already did, keep it 😄